astro_2.0/features/ViewBounty/components/BountyCard/BountyCard.tsx

```diff
@@ -206,117 +211,138 @@ export const BountyCard: React.FC<BountyCardProps> = ({
   }
 
   return (
-    <div
-      tabIndex={0}
-      role="button"
-      onKeyPress={handleCardClick}
-      className={cn(styles.root, {
-        [styles.clickable]: canNavigateToBounty,
-      })}
-      onClick={handleCardClick}
+    <Box
+      bgColor="white"
+      borderRadius="0 8px 8px 0"
+      boxShadow="3px 2px 24px var(--chakra-colors-blackAlpha-300)"
+      cursor="pointer"
+      p={{ base: '22px 20px', sm: '22px 30px' }}
+      sx={{
+        '&:hover': {
+          'box-shadow': '3px 2px 24px var(--chakra-colors-blackAlpha-400)',
+        },
+        transition: 'box-shadow 0.3s ease-out',
+      }}
     >
-      {loading && (
-        <div className={styles.signingTransactionState}>
-          <LoadingIndicator />
-          {t('proposalCard.signingTransaction')}
-        </div>
+      {showBountyTags && (
+        <BountyEditor bounty={bounty} permissions={permissions} />
```

Comment:
Github shows a lot of changes in this file, but mostly are just auto format. These lines are the major changes.